src/HttpServer.h

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include "ThreadPool.h"
+
+using namespace std;
```

Comment:
Please do not use "using namespace". Reference all classes by their full namespace path or add "using" in the class definition.
https://google.github.io/styleguide/cppguide.html#Namespaces